src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -249,13 +249,14 @@ void GcsPlacementGroupManager::OnPlacementGroupCreationSuccess(
   RAY_LOG(INFO) << "Successfully created placement group " << placement_group->GetName()
                 << ", id: " << placement_group->GetPlacementGroupID();
   placement_group->UpdateState(rpc::PlacementGroupTableData::CREATED);
+  // Mark the scheduling done firstly.
+  MarkSchedulingDone();
```

Comment:
Is it possible to add a unit test for this?

Reply:
hmmm... I thought this, but I can't think of any idea to test this even in cpp,  do you have any ideas?

Reply:
I guess it could be difficult without a mock redis